certbot/plugins/manual.py

```diff
@@ -43,7 +43,9 @@ class Authenticator(common.Plugin):
 
 {validation}
 
-Before continuing, verify the record is deployed."""
+Before continuing, verify the DNS record is deployed.
+One way to do this is running the following command at a terminal prompt: nslookup -type=TXT _acme-challenge.{domain}
```

Comment:
Doesn't `{domain}` already include the `_acme-challenge` prefix?

Reply:
true. removed it.